ui/desktop/electron-app/src/services/client-agent-daemon-manager.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+const { spawnSync, spawn } = require('../helpers/spawn-promise');
+
+class ClientAgentDaemonManager {
+  /**
+   * Checks the status of the client agent daemon.
+   * @returns {string}
+   */
+  status() {
+    const clientAgentStatusCommand = ['ferry', 'status', '-format=json'];
```

Comment:
FYI the `boundary ferry` command has been renamed `boundary client-agent`. Flags and environment variables are also renamed accordingly. `boundary ferry` will not work with `0.16.1`.

Reply:
Yup, I'm planning on renaming those in a separate PR to handle the client daemon to `cache` as well. My current build still uses `ferry` so I kept it in, I can add a TODO to make it more explicit

Reply:
No worries, just wanted to make sure we'd communicated it! As you were :)

Reply:
I had no idea about the rename, so good to know 😄